src/main/java/repository/CarHistory.java

```diff
@@ -0,0 +1,21 @@
+package repository;
+
+import java.util.List;
+
+import domain.Car;
+
+public class CarHistory {
+
+    public List<Car> savedCarHistory;
+
+    public CarHistory() {
+    }
+
+    public void saveCarHistory(List<Car> carList) {
+        savedCarHistory = carList;
+    }
```

Comment:
```suggestion
    public CarHistory(List<Car> cars) {
        this.cars = cars;
    }
```
사용하지 않는 코드는 항상 최소화하는게 좋다고 생각합니다~
추가로 CarHistory가 이동할때마다 저장하는 역할보단, 단순히 여러 개의 Car를 들고있는것 같아요
Cars와 같은 일급컬렉션으로 활용해보시는건 어떨까요?

Reply:
Cars라는 도메인을 새로 만들었습니다